src/configuration.ts

```diff
@@ -475,6 +475,7 @@ export function loadAutomationConfig(cfgPath?: string): Configuration {
             cfg = require(cfgPath).configuration;
             cfgLog("automation config");
         } catch (e) {
+            logger.error(e.stack);
```

Comment:
Would it be better to modify the message of the existing Error, `e`, and rethrow that?

Reply:
I don't know. If we did that, we'd need to make whatever catches it and prints it show the stack trace.
This right here is a big improvement over not having the stack trace at all.
I hit this again now, and I really need it.